src/Database/IdentifierQuoter.php

```diff
@@ -180,7 +180,7 @@ protected function _basicQuoter(array $part): array
     {
         $result = [];
         foreach ($part as $alias => $value) {
-            $value = !is_string($value) ? $value : $this->quoteIdentifier($value);
+            $value = is_string($value) ? $this->quoteIdentifier($value) : $value;
```

Comment:
This is the only one I would keep imo

Reply:
Ok so should i revert the changes and keep only this change?

Reply:
Let's see what others think
For me the first case is the happy case usually

Reply:
> For me the first case is the happy case usually

That's fair enough as a personal preference. From a programmatic POV avoiding unneeded negations is better, which I prefer too. Negations also add additional cognitive load.

So let's see what others have to say.

Reply:
Yeah
But !empty in php is basically a classic positive like isset :) well known imo

Reply:
But we dislike use of `empty()` itself don't we? :smile:

Reply:
No not at all if needed :)
It just ain't needed on declared variables